tests/test_views.py

```diff
@@ -359,6 +359,7 @@ def test_move_list_item(self):
             .values_list("sortorder", flat=True),
             [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
         )
+        self.assertEqual(self.list1.list_items.filter(parent=None).count(), 6)
```

Comment:
Before, this was 10, because one of the two lists (the one being moved) had 4 child items, and all of that info was lost. I guess this would be slightly more clear if wrote this to assert against 4 items....